activities.json

```
@@ -1476,11 +1476,11 @@
     "id": "webmidi",
     "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/API/MIDIAccess",
     "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=836897",
-    "mozPosition": "under consideration",
-    "mozPositionDetail": null,
+    "mozPosition": "positive",
+    "mozPositionDetail": "This specification is a reasonable technical design for exposing MIDI devices to JavaScript, which has some demonstrated use-cases in the market. MIDI devices are not universally hardened against adversarial input (especially when sysex is involved) so we don't believe this API is safe to expose in the casual and low-trust context of ordinary websites. We therefore only expose it in Firefox if the user has installed a site-specific add-on to explicitly extend the base web platform with high-trust capabilities ordinarily reserved for installed software.",
```

```suggestion
    "mozPositionDetail": "This specification is a reasonable technical design for exposing MIDI devices to JavaScript, which has some demonstrated use-cases in the market. MIDI devices are not universally hardened against adversarial input (especially when sysex is involved) so we don't believe this API is safe to expose in the casual and low-trust context of ordinary websites. We therefore only expose it in Firefox if the user has deliberately installed a site-specific add-on to enable the capability.",
```

I don't like "high-trust" as a term in general and I think that it is sufficient to just say that we are requiring an add-on.

---

This is better language.  

As big a booster as I am of Web MIDI - and I know you know I am :) - I believe you're doing a disservice to Firefox's users if the UI they are presented with equates Web MIDI access with "installed-software-level access", to the point where I'd even suggest you would serve your users better by not implementing Web MIDI.  You users will then become inured to the scary dialogs, and simply be trained to click OK.  (We saw this with ActiveX controls in IE back in the day.) In particular, MIDI vendors will be explicitly training your users to enable this access, telling them it's OK - which it may be in their context, but the nuance will be lost, and a user will simply say to themselves "oh yeah, it's just like my DAW site".